panel_grid.py

```diff
@@ -48,14 +48,18 @@ def create_panel(self):
         table.pack(fill="both")
 
         self.show_table(self.t.timeline["week" + str(self.week)], table)
+        
 
         week_modifs = Frame(tab)
         week_modifs.pack(side=BOTTOM)
-        next_week = Button(week_modifs,text='next',command= lambda: self.next_week(table))
+        next_week = Button(week_modifs,text='Next Week',command= lambda: self.next_week(table))
```

Comment:
1st bug to look at : 
a) initialise panel_grid like usual
b) add an activity (optional: check a few boxes)
c) click on next week
d) click on last week

_c_ and _d_ create this wierd lasting labels and checkboxes
Your mission, should you choose to accept it: **Clean it up !**